SU-SWS/stanford_news
# D8CORE-000: Changed headline field to 180 characters.

# READY FOR REVIEW 

# Summary
- Changed max chars on headline field

# Needed By (Date)
- Asap

# Urgency
- High

# Steps to Test

1. Check out this branch and import the new config from this module.
2. Validate that the headline field on news items is 180 chars and not 70.

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_profile/pull/207

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
